Pod/Classes/PagingMenuController.swift

```diff
@@ -469,7 +469,7 @@ public class PagingMenuController: UIViewController, UIScrollViewDelegate {
     
     private func currentPagingViewPosition() -> PagingViewPosition {
         let pageWidth = contentScrollView.frame.width
-        let order = Int(ceil((contentScrollView.contentOffset.x - pageWidth / 2) / pageWidth))
+        let order = pageWidth <= 0 ? 0 : Int(ceil((contentScrollView.contentOffset.x - pageWidth / 2) / pageWidth))
```

Comment:
No. This part fixes a crash that occurs in the application. The other fixes a visual bug in the iPad. As I said, we are using our fork. I found it interesting to share.